src/rust/multi.myreadingmanga/res/source.json

```diff
@@ -0,0 +1,57 @@
+{
+	"info": {
+		"id": "multi.myreadingmanga",
+		"lang": "multi",
+		"name": "MyReadingManga",
+		"version": 1,
+		"url": "https://myreadingmanga.info/",
+		"nsfw": 2
+	},
+	"_languages": [
```

Comment:
is there a way to add support for filtering by language or no?

Reply:
yes, but the filter of the site is **AND** not **OR**. if users select multiple languages, it’ll filter out many manga they might actually want. i don’t have android phone, so i’m not sure how tachiyomi does, but [it seems to apply the above filter](https://github.com/tachiyomiorg/tachiyomi-extensions/blob/c6722d44b0e32bb3056f7f9f50884868541b2b1d/src/all/myreadingmanga/src/eu/kanade/tachiyomi/extension/all/myreadingmanga/MyReadingManga.kt#LL75C14-L75C14). 

the search field can exclude keywords users don’t want, but i couldn’t find the full list of languages the site have. plus, some manga have multiple languages in a single manga, using exclude method will also filter them out, e.g. a manga is [Eng/JP], if i don’t select japanese (exclude japanese), this manga will be filtered out. 

i still can’t figure out a better way to filter by language

Reply:
ah, it has an enforce language filter. In that case, I would probably add the same filter (which does the strict AND filtering) in filters.json then and ignore the languages thing in source.json.